spec/features/schedules_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+feature Schedule do
+  let!(:conference) { create(:conference) }
+  let!(:venue) { create(:venue, conference: conference) }
+
+  describe 'with unalphabetized rooms' do
```

Comment:
I don't think we need a spec for `.order`. The `ActiveRecord` test suite will make sure that this will never break...

CI time is developer time.